framework_tvb/tvb/interfaces/web/templates/jinja2/new_footer.html

```diff
@@ -0,0 +1,43 @@
+<!--! ========================================================== -->
```

Comment:
Ideally we would see changes in the prev footer.html file, for being able to compare code

Reply:
Updated the footer.html file